### ISSUE TITLE
[KEYBOARD]: Map needs explicit instructions and keyboard controls that don't interfere with assistive devices

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#514]
## Issue
Facility locator map has several instruction and keyboard shortcut challenges. These were categorized into one issue, and will be summarized here. There are three parts to this issue:

1. The map needs explicit instructions explaining the keyboard shortcuts. As I read the report, these instructions need to be available to all users, so may need to be printed to the page, or in a modal, accordion, etc. Tagging @emilywaggoner for a consult on this one.
2. The `<a>` tags used for zooming in and out need to be buttons and have clear aria-labels describing their function.
3. The keyboard shortcuts `+` and `-` should be removed because they interfere with screen reader native behavior.

This was noted as a SC 2.1.1 defect. Screenshot attached below.

## Audit Finding
* Note 2, Defect 1 of 1

## Acceptance Criteria
As a user, I would like the map to provide clear instructions on how to interact with it. I would expect the instructions to mention the zoom in / zoom out buttons, and the arrow keys. The plus and minus single key shortcuts should no longer change the map.

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/find-locations/

## Possible Remediation
```html
<button
  aria-label="Zoom map in"
  class="leaflet-control-zoom-in"
>
  <span class="sr-only">+</span>
</button>
```

## WCAG or Vendor Guidance (optional)
* [Keyboard: Understanding SC 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)
* [JAWS keyboard shortcuts](https://support.freedomscientific.com/Content/Documents/Manuals/JAWS/Keystrokes.pdf)

## Screenshots or Trace Logs
![Screen Shot 2019-07-08 at 9.31.33 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/50c3e5c4-f926-4848-b0a2-a7fde959e4bf)

---
![Screen Shot 2019-07-08 at 9.59.36 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/98afc187-09a8-40f3-9f8d-1398b4afb363)

**Comment by kelsonic** [comment]
@Karak888 Unassigning myself from this ticket until we get further direction for facility locator :)

**Comment by mmiddaugh** [comment]
This acceptance criteria has been moved to [#4324] and this issue will be closed.
